charts/mirrorbits/tests/custom_values_test.yaml

```diff
@@ -42,10 +63,6 @@ tests:
           value: "RELEASE-NAME-files"
   - it: should define a customized "files" deployment
     template: deployment.files.yaml
-    set:
```

Comment:
Do these tests modifications need to be in this PR? If not, should be in a distinct one IMO.

Reply:
Not completly, but the unit tests were spreading values and it started to be hard to know which suites in the custom values was handling which use case. hence the change to ensure my addons were coherent.